src/util/cleanDu/cleanImmediateDus.js

```diff
@@ -0,0 +1,62 @@
+const fn = require('../../config/fieldNames')
+const getIrrelevantDus = require('./getIrrelevantDus');
+const deleteDus = require('./deleteDus');
+const { logLevel } = require('../logger');
+const logDetails = require('../logDetails');
+const p =  require('../../config/paths');
+const getRawData = require('../getRawData');
+const moment = require('moment');
+//const Set = require("./collections/set");
+const preRun = require('../preRun');
+//receives dataSource,identifiers an object with 3 possible fields, ...
```

Comment:
standard form of comment 
![image](https://user-images.githubusercontent.com/62954509/101360633-932f2880-38a6-11eb-83ef-59664d40db7a.png)